presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -224,6 +224,15 @@ QueryContextManager::toVeloxConfigs(
       traceFragmentId = it.second;
     } else if (it.first == SessionProperties::kQueryTraceShardId) {
       traceShardId = it.second;
+    } else if (it.first == SessionProperties::kShuffleCompressionEnabled) {
+      if (it.second == "true") {
+        configs[core::QueryConfig::kShuffleCompressionKind] = std::to_string(
+            static_cast<uint32_t>(velox::common::CompressionKind_LZ4));
```

Comment:
We default to use lz4? Do we want to provide options for compression kind as well?